### PR TITLE
docs: Cleaned up wording, added steps.

### PR DESCRIPTION
**Pull request by IanEff**
build:
busybot:
chore:
ci:
cd:
docs: setup-demo.md
feat: 
fix:
flux:
perf:
refactor:
revert:
style:
test: manual runthrough

### CODE REVIEW

**Review by zalsader — APPROVED** [review]
Thank you for improving this guide!

Why's the PR's root not the main branch?

File: kubernetes-cluster-kubeone/docs/setup-demo.md [hunk 1]
```diff
@@ -1,51 +1,60 @@
...
-In order to be able to follow all the steps, you would need a Hetzner Api token, to get this you will need: 
-- Sign in into the Hetzner Cloud Console 
-- choose a Project
-- go to Security → API Tokens
-- generate a new token.
+To follow this guide, you will need a Hetzner API token.  To generate this you will need to: 
+- Sign in into the [Hetzner Cloud Console](https://console.hetzner.cloud/)
+- Select your project
+- Go to Security → API Tokens
+- Generate a new token
```
Comment (zalsader) [inline kubernetes-cluster-kubeone/docs/setup-demo.md:11]: Don't we just use the one in 1password?

Comment (IanEff) [reply kubernetes-cluster-kubeone/docs/setup-demo.md:11]: To answer the first question, I was messing around with forking in the VSCode interface, & mucked up the branch.  Apologies!

As for the second, I believe this documentation is meant to be customer-facing at some point, so no need for the 1password stuff.

Comment (zalsader) [reply kubernetes-cluster-kubeone/docs/setup-demo.md:11]: Good point

**Comment by zalsader** [comment]
Closed in favor of #19

**Pull request closed without merging**